classes/CartRule.php

```diff
@@ -830,7 +830,7 @@ public function checkValidity(Context $context, $alreadyInCart = false, $display
             }
 
             if ($cartTotal < $minimum_amount) {
-                return (!$display_error) ? false : $this->trans('You have not reached the minimum amount required to use this voucher', [], 'Shop.Notifications.Error');
+                return (!$display_error) ? false : $this->trans('You have not reached the %s minimum amount required to use this voucher', [$context->getCurrentLocale()->formatPrice($minimum_amount, $context->currency->iso_code)], 'Shop.Notifications.Error');
```

Comment:
The tests reported a usecase where this code fails

```
001 Scenario: cart rule with minimum amount doesn't apply if cart total is lower # Features/Scenario/Cart/Adding/CartRule/minimum_amount_cartrule.feature:12
      When I delete product "product1" from cart "dummy_custom_cart"             # Features/Scenario/Cart/Adding/CartRule/minimum_amount_cartrule.feature:42
        Fatal error: Call to a member function formatPrice() on null (Behat\Testwork\Call\Exception\FatalThrowableError)
```

`$context->getCurrentLocale()` can be null in some circumstances